terraform/virtual-machine/main.tf

```diff
@@ -77,10 +66,9 @@ resource "azurerm_network_security_group" "vm_nsg" {
     source_address_prefix      = "*"
```

Comment:
**[tfsec]** <[AVD-AZU-0048](https://aquasecurity.github.io/tfsec/v1.28.1/checks/azure/network/disable-rdp-from-internet/)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Security group rule allows ingress to RDP port from multiple public internet addresses.